native/explorer/src/expressions.rs

```diff
@@ -367,6 +367,14 @@ pub fn expr_last(expr: ExExpr) -> ExExpr {
     ExExpr::new(expr.last())
 }
 
+#[rustler::nif]
+pub fn expr_concat(left: ExExpr, right: ExExpr) -> ExExpr {
+    let left_expr: Expr = left.resource.0.clone();
+    let right_expr: Expr = right.resource.0.clone();
+
+    ExExpr::new(left_expr.append(right_expr, false))
```

Comment:
@philss Oh, I forgot to mention it.
the 2nd arg of append is [`append_chunks`](https://pola-rs.github.io/polars/py-polars/html/reference/api/polars.Series.append.html).
> 
> Prefer extend over append_chunks when you want to do a query after a single append. For instance during online operations where you add n rows and rerun a query.

> Prefer append_chunks over extend when you want to append many times before doing a query. For instance when you read in multiple files and when to store them in a single Series. In the latter case, finish the sequence of append_chunks operations with a rechunk.

I chose `extend` because I think it is a general way.